app/views/layouts/readme.njk

```diff
@@ -1,3 +1,4 @@
+{% set nunjucksHtmlUsageMessage = '**If you’re using Nunjucks macros in production be aware that using  `html` arguments, or ones ending with `Html` can be a [security risk](https://en.wikipedia.org/wiki/Cross-site_scripting).**' %}
```

Comment:
Could we instead link to the Nunjucks documentation: https://mozilla.github.io/nunjucks/api.html#user-defined-templates-warning ?